plugins/node/instrumentation-react-native-navigation/src/utils/hooks/useSpan.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { MutableRefObject, useRef } from 'react';
+import { Span } from '@opentelemetry/api';
+
+export type SpanRef = MutableRefObject<Span | null>;
+
+const useSpan = (): SpanRef => {
```

Comment:
to analyze if it worth to have this file. here I'm just returning a reference that will contain an instance of an Span (not customizing it yet)

Reply:
if we're keeping it I'd suggest maybe renaming to `useSpanRef` to clear it's dealing with a `SpanRef` rather than a span directly